main.go

```diff
@@ -107,7 +107,12 @@ func main() {
 				log.Hint("    %s", uErr.Hint)
 			}
 		}
-
+		if cErr, ok := err.(errors.CommandError); ok {
+			if strings.Contains(cErr.Reason.Error(), "exit code 137") || strings.Contains(cErr.Reason.Error(), "4294967295") || strings.Contains(cErr.Reason.Error(), "status 137") {
+				log.Yellow(`Insufficient memory. Please update your resources on your manifest.
```

Comment:
Insufficient memory. Please update your resources on your manifest.
```suggestion
				log.Yellow(`Insufficient memory. Please update your resources on your okteto manifest.
```